identity-apps-core/pom.xml

```diff
@@ -563,6 +563,25 @@
                 <artifactId>org.wso2.identity.apps.taglibs.layout.controller</artifactId>
                 <version>${project.version}</version>
             </dependency>
+            <!-- Testing related dependencies -->
+            <dependency>
+                <groupId>org.testng</groupId>
+                <artifactId>testng</artifactId>
+                <version>${testng.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.mockito</groupId>
+                <artifactId>mockito-core</artifactId>
+                <version>${mockito.version}</version>
+                <scope>test</scope>
+            </dependency>
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>${junit.version}</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
why junit is required ?

Reply:
In a previous commit I had used JUnit Annotations instead of TestNG Annotations...I will remove this since it's not needed now.

Reply:
This is now removed.